src/TesterInternal/TimerTests/ReminderTests_AzureTable.cs

```diff
@@ -300,12 +287,12 @@ public void TestCleanup()
             Task<bool> t1 = Task.Factory.StartNew(() =>
             {
 
-                StopSilo(siloToKill);
+                siloHost.StopSilo(siloToKill);
```

Comment:
Every time i see the `siloHost` i get confused about what it is!
1) it seems like it should have a capital 'S' to stop it looking so much like a private field in this class - it's a "property" shared / inherited from the base class, isn't it? From the name, i don't know where to expect it to "live".
2) isn't that object a "handle" wrapping an in-process mini-cluster of multiple silos, rather than just a single "silo"? Would a name along the lines of `HostedTestCluster.StopSilo(x)` make that clearer?
3) There seems to be a natural evolution from this "single cluster" to the "multi cluster" tests what @sebastianburckhardt is working on. Will we re-use this "cluster host" directly for the multi-cluster cases?